src/acceleration/BaseQNAcceleration.cpp

```diff
@@ -367,13 +422,7 @@ void BaseQNAcceleration::performAcceleration(
      * PRECONDITION: All objects are unscaled, except the matrices within the QR-dec of V.
      *               Thus, the pseudo inverse needs to be reverted before using it.
      */
-    Eigen::VectorXd xUpdate = Eigen::VectorXd::Zero(_residuals.size());
-    computeQNUpdate(cplData, xUpdate);
-
-    /**
-     * apply quasiNewton update
-     */
-    _values = _oldValues + xUpdate + _residuals; // = x^k + delta_x + r^k - q^k
```

Comment:
Where did this line go?

Reply:
I also just stumbled over the line `_values = _oldValues + xUpdate + _residuals; // = x^k + delta_x + r^k - q^k` and think it is pretty hard to follow how things are being updated here (btw. what is `q^k`?). Something like `x_kp1 = xUpdate + _values` or just `_values += xUpdate` would be much clearer from my point of view. I just tried it out and according to our tests it seems to work (see https://github.com/precice/precice/pull/1979).